src/volt/components/VoteControls/styles.js

```diff
@@ -30,6 +25,7 @@ export const StyledSlider = styled(Slider).attrs(() => ({}))`
   user-select: none;
   width: 100%;
   min-width: auto;
+  margin-bottom: 8px;
```

Comment:
Maybe bettrer to put inside of attributes `mb:  2`

Reply:
I tried but it doesn't pick it up... any ideas?

Reply:
Did you keep this one here? :)
need to remove it as well. otherwise what you specify in attrs will be overwritten

Reply:
![image](https://user-images.githubusercontent.com/134680/64463209-7afcb680-d103-11e9-8fe4-8c042bd51b76.png)

:woman_shrugging:

Reply:
Right... Screw it then, keep as margin-bottom then.
Just add 
`margin-bottom: ${({theme}) => `${theme.space[2]}px`};`